engine/gamesys/src/gamesys/scripts/script_physics.cpp

```diff
@@ -1753,6 +1894,198 @@ namespace dmGameSystem
         dmScript::TeardownCallback(cbk);
     }
 
+    static void PushCollision(lua_State* L, dmPhysicsDDF::Collision* collision)
+    {
+        lua_createtable(L, 0, 3);
+
+        dmScript::PushVector3(L, *((dmVMath::Vector3*) &collision->m_Position));
+        lua_setfield(L, -2, "position");
+        dmScript::PushHash(L, collision->m_Id);
+        lua_setfield(L, -2, "id");
+        dmScript::PushHash(L, collision->m_Group);
+        lua_setfield(L, -2, "group");
+    }
```

Comment:
Instead of dmScript::PushDDF, we use dedicated functions. It turned out to be 4x faster.
Draw back is: more code, and potential maintainability issues.